textproc/lilypond-devel/Portfile

```diff
@@ -21,59 +22,150 @@ long_description    Lilypond is a unix-based automated engraving system that \
 homepage            http://lilypond.org/
 conflicts           lilypond
 platforms           darwin
-master_sites        http://download.linuxaudio.org/lilypond/sources/v${branch}/
+master_sites        http://lilypond.org/download/sources/v${branch}/
+# master_sites      http://download.linuxaudio.org/lilypond/sources/v${branch}/
 distname            ${my_name}-${version}
-checksums           rmd160  f0d923cec535cf8a133f2c801082db6120f01ad6 \
-                    sha256  3679db30a3f74ce8668691952fb0dd58d8ad6bad099017e815ca3e4312261453
+checksums           rmd160 71f199afa24a75cfde53337c2140e96c74d7857d \
+                    sha256 a5679cddb3e415828642f75027997596b814922aadb1fa633c8f973238ae291b \
+                    size   17407520
 
 depends_build       port:t1utils \
                     port:bison \
                     port:texi2html \
                     port:netpbm \
                     port:pkgconfig \
-                    port:flex
-depends_lib         port:fontconfig \
+                    port:flex \
                     port:fontforge \
+                    port:libtool
+
+depends_lib         port:fontconfig \
                     port:freetype \
                     port:gettext \
                     path:lib/pkgconfig/glib-2.0.pc:glib2 \
                     port:ghostscript \
                     port:gmp \
-                    port:mftrace \
                     port:guile18 \
                     port:texinfo \
                     path:lib/pkgconfig/pango.pc:pango \
-                    port:flex \
-                    port:t1utils \
                     port:texlive \
                     port:texlive-fonts-recommended \
                     port:texlive-lang-cyrillic \
                     port:texlive-metapost \
                     port:dblatex \
-                    port:libtool \
-                    port:python27
+                    port:python27 \
+                    port:urw-core35-fonts
 
+configure.cmd       autoconf -f && ./configure
 configure.python    ${frameworks_dir}/Python.framework/Versions/2.7/bin/python2.7
 
-patchfiles          patch-scripts-build-mf2pt1.pl.diff
+set lilyponddevel.texgyredir \
+    "${prefix}/share/texmf-texlive/fonts/opentype/public/tex-gyre"
+set lilyponddevel.mactex_bin   ""
+set lilyponddevel.temp         ""
+
+variant mactex description {Allow lilypond-devel to use MacTeX or another\
```

Comment:
I don't want to block the merge of this PR (I'm perfectly fine with this PR being merged without addressing this comment), but at this point (with at least two ports using such a convoluted code) it would really make sense to separate this chunk of code into its own PortGroup. It could be the texlive PortGroup if needed, or a separate one, depending on what is easier to implement, but repeating and then maintaining this code all over the place sounds super painful to me. Will you copy the same chunk to `lilypond`?

Reply:
Your suggestion sounds sensible, however, I neither have the time nor the experience to fiddle with PortGroups (and, admittedly, a lack of interest :-) ).

And no, I won't update `lilypond`.  I no longer use this ancient version.  As soon as 2.20 gets released it makes sense to (more or less) copy `lilypond-devel`'s portfile to `lilypond`.

Reply:
I'd see a separate ticket for the PortGroup situation as well.